packages/engine/src/scene/components/LinkComponent.ts

```diff
@@ -37,24 +37,31 @@ export const LinkComponent = defineComponent({
   jsonID: 'link',
 
   onInit: (entity) => {
-    setComponent(entity, InputComponent)
+    try {
+      setComponent(entity, InputComponent) // TODO implement schemas for componenets to use in behave graph
```

Comment:
let's move this to a reactor useEffect with no deps, initializers should be kept to simply specifying the schema